### DISCUSSION TITLE
Report period (date) missing when printing reports

### DISCUSSION THREAD

**Discussion by mbugikp** [Blair2004/NexoPOS#1597] [Ideas]
**Describe the bug**
When obtaining a report we normally fill start date & end date=period however when we print the report this period is not printed, therefore when you are presented with such a printed report you cannot know it represents which period.

**To Reproduce**
Go to reports pick any report e.g. sales report, change start date and end date, print

**Expected behavior**
the printed report should clearly indicate, report period, and print date&time 

**Screenshots**

![missing period date when printing 01](https://user-images.githubusercontent.com/13999977/236900751-b636652d-a9b9-4680-8fa4-8d6fca4cb4b5.png)
start date & end date set (period)
![missing period date when printing 02](https://user-images.githubusercontent.com/13999977/236900418-d78037bf-ff07-42d0-951a-e3e4564bd5ab.png)
report period missing from the printed report

**Environment:**
 - NexoPOS 4.8.17

**Comment by Blair2004** [comment]
Approved for v5
